drivers/peci/peci_ite_it8xxx2.c

```diff
@@ -317,6 +327,13 @@ static int peci_it8xxx2_init(const struct device *dev)
 	/* Initialize Semaphore */
 	k_sem_init(&data->device_sync_sem, 0, 1);
 
+	int ret = clock_control_on(config->clk_dev,
+			       (clock_control_subsys_t *)&config->clk_cfg);
+	if (ret < 0) {
+		LOG_ERR("Turn on clock fail %d", ret);
```

Comment:
`Failed to turn on clock control`?

Reply:
Yup, you're right, and I'll correct it as follows:

`LOG_ERR("Failed to enable clock_control_on(): %d", ret);`